src/hooks/useSnippets.tsx

```
@@ -124,6 +124,11 @@ const fetchSnippets = async ({
 }): Promise<PaginatedResponse> => {
   const actualPageSize = pageSize || 10
 
+  // We won't pass political spectrum when it's unset
+  if (filters && !filters?.politicalSpectrum) {
+    delete filters?.politicalSpectrum
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)